nix_update/__init__.py

```diff
@@ -132,14 +136,29 @@ def nix_build(options: Options) -> None:
     )
 
 
+def nix_test(git_dir: str, package: Package, system: Optional[str]) -> None:
+    if not package.tests or not system:
+        die(f"Package {package.name} does not define any tests for {system}")
+    else:
+        tests = []
+        for t in package.tests:
+            tests.append("-A")
+            tests.append(f"tests.{t}.{system}")
+        cmd = ["nix-build", f"{git_dir}/nixos/release.nix"] + tests
```

Comment:
Not all tests are nixos tests. Sometimes people just reference derivations here. I will have a look at that.

Reply:
Interesting, I don't think I have ever seen that. I basically tried to do something similar to @ofborg here

Reply:
It's an edge case I have seen it only a handful times.